servant-client/test/Servant/StreamSpec.hs

```diff
@@ -78,9 +84,8 @@ api :: Proxy StreamApi
 api = Proxy
 
 getGetNL, getGetNS :: ClientM (SourceIO Person)
-getGetALot :: ClientM (SourceIO BS.ByteString)
 getStreamBody :: SourceT IO BS.ByteString -> ClientM (SourceIO BS.ByteString)
-getGetNL :<|> getGetNS :<|> getGetALot :<|> getStreamBody = client api
+getGetNL :<|> getGetNS :<|> _ :<|> getStreamBody = client api
```

Comment:
```suggestion
getGetNL :<|> getGetNS :<|> _getGetALot :<|> getStreamBody = client api
```

This will suppress the warning while being explicit about what we are ignoring.